### ISSUE TITLE
ARIA: separator role has unrelated ARIA range documentation

### ISSUE THREAD

**Issue by clshortfuse** [mdn/content#25997]
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/separator_role

### What specific section or headline is this issue about?

associated_wai-aria_roles_states_and_properties and specification

### What information was incorrect, unhelpful, or incomplete?

>[aria-valuenow](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-valuenow)
If the separator is focusable and has a known value, the aria-valuenow defines the current value. If not focusable or the value is unknown, do not include this attribute.

>[aria-valuemin](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-valuemin) (default is 0)
If the separator is focusable, and the minimum value is not 0, include the minimum value with aria-valuemin. If the

>[aria-valuemax](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-valuemax) (default is 100)
If the separator is focusable, and the maximum value is not 100, include aria-valuemax with a value equal to or larger than aria-valuemin.

>[aria-valuetext](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-valuetext)
If the separator is focusable, and the aria-valuenow is not optimal in providing the user with usable information, included aria-valuetext content will be read instead of the aria-valuenow value.

-----

>ARIA Authoring Practices
>\# range_related_properties

### What did you expect to see?

Nothing related to aria range

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/web/accessibility/aria/roles/separator_role`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/separator_role
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/accessibility/aria/roles/separator_role/index.md
* Last commit: https://github.com/mdn/content/commit/a12fb8f2612fbdc6635b68c1dcd3c2fce212c10c
* Document last modified: 2023-02-23T10:50:32.000Z

</details>

**Comment by clshortfuse** [comment]
I'm wrong. `separator` has extra states if focusable.
